qiskit_experiments/calibration_management/calibrations.py

```diff
@@ -93,6 +132,270 @@ def __init__(self, control_config: Dict[Tuple[int, ...], List[ControlChannel]] =
         self._hash_to_counter_map = {}
         self._parameter_counter = 0
 
+        self._library = None
+        if library is not None:
+            self._library = library
+
+            # Add the basis gates
+            for gate in library.basis_gates:
+                self.add_schedule(library[gate], num_qubits=library.num_qubits(gate))
+
+            # Add the default values
+            if add_parameter_defaults:
+                for param_conf in library.default_values():
+                    schedule_name = param_conf[-1]
+                    if schedule_name in library.basis_gates:
```

Comment:
Should `library.default_values()` only iterate over the schedules in `library.basis_gates` (making this test unnecessary)?

Reply:
It already does. I think this check is not needed. See here: f75d4ae